### PR TITLE
Update to SoundSwallower 0.6.0

### PR DESCRIPTION
**Pull request by dhdaines**
Allows us to reduce, but sadly not eliminate, custom webpack configuration, because webpack's default behaviour is fine but Angular overrides it internally with something that doesn't work at all: https://github.com/angular/angular-cli/issues/24617

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -19,11 +19,12 @@
                 "@angular/platform-browser": "^15.1.1",
                 "@angular/platform-browser-dynamic": "^15.1.1",
                 "@angular/router": "^15.1.1",
+                "@types/emscripten": "^1.39.6",
```
Comment (joanise) [inline package-lock.json:22]: I was going to ask why we now need this, but it was already in `package-lock.json` via ss.js, so you're just making it explicit.

**Review by joanise — APPROVED** [review]
Looks good, and works on my machine. My only question is whether you're planning to upgrade to ss 0.6.1 soon too, since you've just created that tag, but it doesn't matter, we can merge this and merge a 0.6.1 update later too.

File: package-lock.json (hunk 1 above)
Comment (dhdaines) [reply package-lock.json:22]: we *shouldn't* actually need it, as it is already a dependency from soundswallower, but npm does random things sometimes so it helps to make it explicit

**Comment by dhdaines** [comment]
Ah, actually it would probably be good to go to soundswallower 0.6.1, as there is a small problem in 0.6.0 (which doesn't seem to necessarily cause a failure all the time, not sure why): https://github.com/emscripten-core/emscripten/issues/18610

**Comment by dhdaines** [comment]
So also - there are two possible solutions - in the long run we may want to explicitly include the WASM in `assets` and disable entirely `import.meta.url` parsing by webpack, since this matches what Angular+esbuild will do (even though I consider it to be totally wrong because WASM is not an "asset" but part of the code), but that runs into an Angular bug which breaks testing, so this is the best way to go for the moment.

**Comment by joanise** [comment]
I'm going to trust your judgement on all that. I saw a long conversation between you and Aidan about that, but I have not taken the time to understand it yet.

**Pull request merged**